devicetypes/zooz/zooz-zen52-double-relay.src/zooz-zen52-double-relay.groovy

```diff
@@ -0,0 +1,437 @@
+/*
+ *  Zooz ZEN52 Double Relay
+ *
+ *  Changelog:
+ *
+ *    2022-03-02
+ *      - Publication Release
+ *
+ *  Copyright 2022 Zooz
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+*/
+
+import groovy.transform.Field
+
+@Field static Map commandClassVersions = [
+	0x20: 1,	// Basic
+	0x22: 1,	// ApplicationStatus
+	0x25: 1,	// SwitchBinary
+	0x55: 1,	// TransportService
+	0x59: 1,	// AssociationGrpInfo
+	0x5A: 1,	// DeviceResetLocally
+	0x5B: 1,	// CentralScene
+	0x5E: 2,	// ZwaveplusInfo
+	0x60: 3,	// MultiChannel
+	0x6C: 1,	// Supervision
+	0x70: 1,	// Configuration
+	0x72: 2,	// ManufacturerSpecific
+	0x73: 1,	// Powerlevel
+	0x7A: 2,	// FirmwareUpdateMd
+	0x85: 2,	// Association
+	0x86: 1,	// Version
+	0x87: 2,	// Indicator (3)
+	0x8E: 2,	// MultiChannelAssociation
+	0x98: 1,	// Security S0
+	0x9F: 1		// Security S2
+]
+
+@Field static int supervisionCC = 108
+@Field static int btnPushed = 0
+@Field static int btnReleased = 1
+@Field static int btnHeld = 2
+@Field static int mainEndpoint = 0
+@Field static List<Integer> relayEndpoints = [1, 2]
+@Field static List<String> supportedButtonValues = ["pushed","held","pushed_2x","pushed_3x","pushed_4x","pushed_5x"]
+
+@Field static Map configParams = [
+	ledIndicator: [num:2, title:"Led indicator", size:1, defaultVal:1, options:[0:"Disabled", 1:"Enabled"]],
+	sceneControl: [num:16, title:"Scene Control", size:1, defaultVal:1, options:[0:"Disabled", 1:"Enabled"]],
```

Comment:
How does this parameter work in case of this device ? Does it affect CentralSceneNotification in any way ?

Reply:
Disabling that setting would break the button functionality so I've removed it.